src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -11,7 +11,8 @@
     >
       <button
         :value="item.value"
-        :class="{ disabled, selected: item.selected }"
+        :class="{ selected: item.selected }"
+        :disabled="item.disabled"
```

Comment:
I think this needs to be updated to prevent `disabled="false"`
```suggestion
        :disabled="item.disabled === true ? true : undefined"
```